### ISSUE TITLE
"Full export note" not working

### ISSUE THREAD

**Issue by mpenguin** [argenos/zotero-mdnotes#208]
**Describe the bug**
Using mdnotes in Zotero, I can "export to markdown" no problem - it creates the file and I can see see it in both Zotero and Obsidian. However, the "create full export note" does nothing - no file is created, nothing is in Zotero, Obsidian, or File Explorer.

**To Reproduce**
1. Go to 'tools' > 'mdnotes'
2. Click on 'create full export note'
3. See that nothing happens

**Expected behavior**
I expect it to create a full export note. The "export to markdown" creates a note according to the template, but "create full export note" does not create anything. I'm using Bryan Jenks' templates in Obsidian. I know these work - he's made videos on them and they are available publically.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 116.0.2.0
 - Mdnotes version: 0.2.3
 - Zotfile version: 5.1.2
 - BetterBibtex version: 6.7.54

**Additional context**
I get the following output from Zotero debut log when attempting to create a full export note:

```
[(3)(+0003630): Getting contents of C:\Users\user\OneDrive\ZettleKasten\My Brain\Templates\mdnotes\Mdnotes Default Template.md

(3)(+0000011): Getting contents of C:\Users\user\OneDrive\ZettleKasten\My Brain\Templates\mdnotes\Zotero Note Template.md

(3)(+0000004): TypeError: R is not a constructor e[155]/B.prototype.parseFromString@chrome://mdnotes/content/markdown-utils.js:2:97646 I@chrome://mdnotes/content/markdown-utils.js:2:98302 turndown@chrome://mdnotes/content/markdown-utils.js:2:100931 Zotero.MarkdownUtils</this.html2md@chrome://mdnotes/content/markdown-utils.js:2:149448 noteToMarkdown@chrome://mdnotes/content/mdnotes.js:340:24 getZoteroNoteFileContents@chrome://mdnotes/content/mdnotes.js:722:14]
```

**Comment by wongsanri** [comment]
I encountered the same problem, but now it is all settled. I was unable to "create full export note". 
And I think this plug-in may conflict with some other plug-ins, so I disabled all the others and then tried to "create full export note", and it worked magically. 

And then, I turn on the plug-in once at a time and test whether it still works. It turns out that it conflicts with a plug-in named "pubpeer_zotero_plugin". 

Hope this will help you! If not, I have no idea.
